### PR TITLE
Updating tests to use the correct targetType for new plugins

### PR DESCRIPTION
**Pull request by teone**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,7 +21,7 @@ jenkins-test: build deps license_check linters
 	TEST_PACKAGES=github.com/onosproject/onos-config/... ./build/build-tools/build/jenkins/make-unit
 
 helmit-config: integration-test-namespace # @HELP run helmit gnmi tests locally
-	helmit test -n test ./cmd/onos-config-tests --suite config
+	helmit test -n test ./cmd/onos-config-tests --suite config -c ../../onos-helm-charts
```
Comment (adibrastegarnia) [inline Makefile:24]: I think for integration tests we use the published charts. Any reason we should use local charts in the make file?

File: test/utils/charts/umbrella.go [hunk 2]
```diff
@@ -16,12 +16,13 @@ package charts
 
 import (
 	"github.com/onosproject/helmit/pkg/helm"
-	"github.com/onosproject/onos-test/pkg/onostest"
 )
 
+const localRepo = "/home/teone/Sites/onos-helm-charts"
```
Comment (adibrastegarnia) [inline test/utils/charts/umbrella.go:21]: Yeah you should remove this

File: Makefile (hunk 1 above)
Comment (teone) [reply Makefile]: my bad, didn't intend to commit that

File: test/utils/charts/umbrella.go (hunk 2 above)
Comment (teone) [reply test/utils/charts/umbrella.go]: as for the other comment

**Review by adibrastegarnia — APPROVED** [review]

**Pull request merged**